dataset/dictionary/living-beings.json5

```diff
@@ -293,10 +293,41 @@
     pronunciationJa: "かれい",
     tags: [ "living-being", "sumeru" ],
   },
+  {
+    en: "Khvarena",
+    ja: "霊光 / 霊光百種",
+    zhCN: "灵光 / 灵光百种",
```

Comment:
@Bill-Haku @SleepyAsh0191 Can you review if these Chinese translations are correct when you have time?

Reply:
I didn't found 灵光百种 in the internet. Also I don't have an impression of this word when I was playing the new world event. Maybe I need to check it in the game tonight. Can you give me more details about this word?

Reply:
@Bill-Haku Thanks, in Japanese text, 霊光百種 may mean the number of Khvarena. 
I will look for the text including 霊光百種 and capture a screenshot if I find it.

Reply:
thanks for your explanation. your screenshot will help a lot.

Reply:
I found Zurvan mentioned this word.

![cn-1](https://user-images.githubusercontent.com/107226297/232357346-29c27a45-9ed9-4bde-ad24-e79de56c50dd.png)
![en-1](https://user-images.githubusercontent.com/107226297/232357363-8a2fd041-4c5c-40b6-8f36-c46eb70f9ae1.png)
![ja-1](https://user-images.githubusercontent.com/107226297/232357368-3040608f-f047-4d25-9cd7-1cadc4494d67.png)

---

I found there is the English translation for 霊光百種 / 灵光百种 too, so I will separate 灵光 and 灵光百种.

![cn-2](https://user-images.githubusercontent.com/107226297/232357361-69b694cf-a2a9-4438-bac7-f4641f1c192d.png)
![en-2](https://user-images.githubusercontent.com/107226297/232357367-32a6fa73-555a-40c6-b674-74b4c26d6f36.png)
![ja-2](https://user-images.githubusercontent.com/107226297/232357372-938722f5-1185-4557-a7e6-1ab683a6370a.png)

Reply:
Thanks for your confirmation. The words is correct. 

BTW, I think in the common word order, the word should be 百种灵光. However, it is common to put the attribute behind the noun when the author wants to emphasize the attribute in ancient Chinese articles or some words in modern Chinese. So I'd consider 灵光百种 as a phrase rather than a single word.

Reply:
@Bill-Haku Thanks.

> So I'd consider 灵光百种 as a phrase rather than a single word.

Japanese 霊光百種 sounds like a single word and a proprietary noun, but English "the countless motes of Khvarena" sounds the phrase to me. It does not sound like a proprietary noun either.
Maybe the English expression is a better translation than the Japanese one.

Since I also add phrases in the dictionary if they are worth adding, it is not a problem to add 灵光百种 even if it is a phrase. However, since Genshin Dictionary is a dictionary for Genshin's proprietary nouns in general (while it is not so strict policy and some non-proprietary nouns such as Qilin and Shogunate are already registered), we may have to consider if 灵光百种 is still worth adding.

For now, I merge this PR but feel free to add comments to this thread if you have any comments for this issue.

Reply:
I just give you my understanding of this word, and it depends on you to keep it in the dictionary or not.

The English text is the most simple expression. Zurvan says that the Pari is the only one among all kinds of Khvarena. However, mihoyo didn't use a common expression of this word in Chinese and Japanese, which should be  各种灵光 or 様々な霊光, they wanted to use a word that sounds more literary, 百种灵光, and 百 doesn't means hundred here, it just means many. Although it is a usage of ancient Chinese, it's still common in modern Chinese, but maybe not common in modern Japanese. In my experience, almost all the words similar to it comes from Chinese. So you may consider it as a proprietary noun. Also, mihoyo reversed the order of the words to 灵光百种, it sounds more likely to be a proprietary noun. In fact, I didn't fully understand the meaning or the reason of using this word until I read the English text.

Reply:
Thanks for your insights!
I understand 灵光百种 sounds like a proprietary noun for Chinese speakers too. Then I think we can keep this word in the dictionary.

> 百 doesn't means hundred here, it just means many, all kinds of, just as 各.

Thanks for the explanation. There's similar expression in Japanese too. In Japanese, we usually use 千 (thousand) or 万 (ten thousand) to express "many" instead of 百. (百 sounds too small to express "many" for Japanese speakers.)
e.g.
- 千客万来 (せんきゃくばんらい / Senkyaku Banrai) ― So many customers come
- 万国 (ばんこく / Bankoku) ― various countries

Reply:
yes, your examples are good. thanks for your decision. btw, in fact, even 三 can has the meaning of `many` or `all kinds` in some words, and also 十 or 百 or 千... I cannot understand the logic of the ancestors lol.